### PR TITLE
authn v2 support

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

**Comment by r3dcrosse** [comment]
@pangea-andrest in `authn/clients/ClientPassword.java` it's still using a v1 endpoint

File: packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/authn/requests/FlowStartRequest.java [hunk 1]
```diff
@@ -38,7 +36,7 @@ public static class Builder {
 
 		private String cbURI;
 		private String email;
-		private FlowType flowType;
+		private FlowType[] flowType;
 		private IDProvider provider;
```
Comment (r3dcrosse) [inline packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/authn/requests/FlowStartRequest.java:40]: The v2 endpoint no longer takes an IDProvider. Missing an invitation field here now:
Screenshot 2023-10-19 at 5 44 26 PM

**Comment by r3dcrosse** [comment]
@pangea-andrest I pushed some commits to add/update doc strings. Please pull before making any commits.

File: packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/authn/requests/FlowStartRequest.java (hunk 1 above)
Comment (pangea-andrest) [reply packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/authn/requests/FlowStartRequest.java:40]: fixed! Thanks!

**Review by r3dcrosse — APPROVED** [review]

**Pull request merged**